.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,48 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [dask-gateway, dask-gateway-server]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
These are some hard pins. Perhaps it would be ok to relax this to `docker/login-action@v1`

Reply:
Note I think dependabot translates these bumps of tags to hashes no matter what in the end if dependabot is configured to keep the feferenced github actions updated.

If it references a branch, its different though i think.

Reply:
I'll address this via dependabot configuration to bump actions.